docs/creating-manifest/events.md

```diff
@@ -1757,7 +1757,93 @@ onAfterRemoveSSL:
 ```
 @@!
 
+### onBeforeInstallAddon  
+The event is executed before add-on installation  
 
+Event Placeholders:  
+
+-   `${event.params.}`:  
+    - `settings` - custom form with predefined user input elements  
+    - `id` - unique add-on id  
+    - `lang` - [user's language code](https://en.wikipedia.org/wiki/Language_code) which was used during the installation. Optional parameter  
+    - `region` - region, where an environment will be installed  
+    - `targetAppid`  - application identifier  
+-   `${event.response.}`: parameters are absent  
+
+### onAfterInstallAddon  
+The event is executed after add-on installation  
+  
+-   `${event.params.}`:  
+    - `settings` - custom form with predefined user input elements   
+    - `id` - unique add-on id  
+    - `lang` - [user's language code](https://en.wikipedia.org/wiki/Language_code) which was used during the installation. Optional parameter  
+    - `region` - region, where an environment will be installed  
+    - `targetAppid`  - application identifier  
```

Comment:
change to "unique environment *appid* at the Jelastic Platform"

Reply:
done:
https://github.com/vlobzakov/docs/pull/19/commits/794d0b8d68d7c3a24eddcc5433116766cc222e61
https://github.com/vlobzakov/docs/pull/19/commits/1ffa9f07646e16b7a20380bc3f4a29c34db5067b